aip/aog/3022.md

```diff
@@ -156,7 +156,7 @@ following Schema.org entity types to URI parameters:
 <!-- prettier-ignore-start -->
 [aip-3021]: ./3021.md
 [date]: https://github.com/googleapis/api-common-protos/tree/master/google/type/date.proto
-[datetime]: https://github.com/googleapis/api-common-protos/tree/master/google/type/datetime.proto
+[datetime]: https://github.com/googleapis/googleapis/blob/master/google/type/datetime.proto
```

Comment:
I've also changed tree -> blob (which I believe is slightly more correct for files). Obviously haven't attempted to do so throughout.